docs/index.rst

```diff
@@ -1,11 +1,75 @@
-Girder: High-Performance Data Management
-========================================
+Girder: A data management platform
+==================================
 
-For a high-level overview of the system, we recommend checking out the :doc:`user-guide`.
+What is Girder?
+---------------
+
+Girder is a free and open source web-based **data management platform**. What does
+that mean? Girder is both a standalone application as well as a platform on top of
+which new web services can be quickly and easily constructed. It's meant to be a
```

Comment:
Girder is both a standalone application and a platform for building new web services.  It's meant to allow quick and easy construction of web-based applications that have some or all of the following requirements: